packages/smart/contracts/turbo/Grouped.sol

```diff
@@ -9,10 +9,10 @@ import "./GroupFetcher.sol";
 abstract contract Grouped is AbstractMarketFactoryV3, CalculateLinesToBPoolOdds {
     event GroupCreated(uint256 indexed id, uint256 endTime, uint256 invalidMarketId, string invalidMarketName);
     event GroupMarketAdded(uint256 indexed groupId, uint256 marketId, string marketName);
-    event GroupReady(uint256 indexed id);
+    event GroupFinalizing(uint256 indexed groupId, uint256 winningMarketIndex);
     event GroupResolved(uint256 indexed id, bool valid);
 
-    enum GroupStatus {Unknown, BeingCreated, Scheduled, Final, Invalid}
+    enum GroupStatus {Unknown, Scheduled, Finalizing, Final, Invalid}
```

Comment:
When would it be finalizing?

Reply:
"Finalizing" means the markets are being resolved. It prevents the controller from beginning the finalization process twice for the same group, which could cause a weird state like multiple markets resolving as Yes, as well as preventing adding new markets when finalization begins.

When finalization starts, it should be finished as soon as possible, so you can't do much with the market factory once it starts.